content/en/downloads/_index.md

```diff
@@ -9,14 +9,36 @@ images: []
 weight: 100
 layout: default
 ---
-The narayana-full zip is not downloadable anymore from the website, but you can still download single components from [maven central](https://mvnrepository.com/artifact/org.jboss.narayana), build it from github (i.e. [7.2.2.Final](https://github.com/jbosstm/narayana/tree/7.2.2.Final)) or download it from github tag (i.e [7.2.2.Final](https://github.com/jbosstm/narayana/releases/tag/7.2.2.Final))
+The narayana-full zip is not downloadable anymore from the website, but you can still download single components from [Maven Central](https://mvnrepository.com/artifact/org.jboss.narayana), build it from github (i.e. [7.2.2.Final](https://github.com/jbosstm/narayana/tree/7.2.2.Final)) or download it from github tag (i.e [7.2.2.Final](https://github.com/jbosstm/narayana/releases/tag/7.2.2.Final))
 
 
 | | Version | Date | License |
 | ---------------------------------------- | ----------- | ----------------- | ---------------- |
 | [Narayana Source Code](https://github.com/jbosstm/narayana/releases/tag/7.2.2.Final) | 7.2.2.Final | 2025-05-12 | Apache-2.0
 
-[Legacy Narayana versions](../downloads-other/index.html)
+### GPG Public Key
+
+Starting with version 7.2.3.Final, all Narayana [Maven artifacts](https://mvnrepository.com/artifact/org.jboss.narayana) are signed with JBossTM GPG key.
+The corresponding public key is:
+
+**ID**: rsa4096/F54B74513B168DB7 2025-07-11\
+**Fingerprint**: 3AA1 3E1B 77D4 1EEC 473E  1FAF F54B 7451 3B16 8DB7\
+<a href="/files/publicKey.gpg" download>**Download jbosstm's Public Key**</a>
+
+To verify the authenticity of JBossTM Maven artifacts, download both the `.jar` file and its corresponding `.asc` signature file from Maven Central.
+Then, import the above public GPG key using
+```bash
+gpg --import publicKey.gpg
```

Comment:
Is there a way to do it without the user importing the but maybe pointing at it? If we tell them to import it, we need to tell them to delete it too in case we cycle the key...

Reply:
I am not sure it's possible from what I can see quickly. So maybe we just need to add a "warning"|"note" that it's important for users to make sure to check the current Narayana public key from our website is the one imported into their keyring before verifying files each time (deleting old public keys from their keyring first)?

Reply:
Done